content/docs/how-we-work/security.md

```diff
@@ -10,6 +10,29 @@ Security is an everlasting concern in the world of software engineering. Underst
 
 That said, security is a wide and deep topic and a single person cannot hope to span the width and depth of this challenging area. It is the team's responsibility to understand security at various levels and implement safeguards.
 
-For example, an application relies on database credentials to connect to the database. There can be API keys using which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them.
+For example, an application relies on database credentials to connect to the database. There can be API keys using which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them. Use the environment variables or vaults for storing and retrieving them.
 
-Use the environment variables or vaults for storing and retrieving them.
+## Server and Website Security
+
+In particular, we practice server and site security by using only sanitized databases, taking care not to install restricted access files on development or personal instances outside the project defined security accreditation boundary and scrubbing unneeded data from our development systems
+And as developers, we follow technology specific coding standards and best practices and write secure code as per the respective community’s guidelines.
```

Comment:
Is this line part of the previous paragraph? If so, then keep it consistent with the rest of the text and include it on the same line. If not, please put in a line break so that it actually renders as a separate paragraph.